demos/serverless/src/handlers.js

```diff
@@ -208,3 +208,55 @@ exports.end = async(event, context, callback) => {
   }).promise();
   callback(null, response);
 };
+
+exports.logs = async (event, context) => {
```

Comment:
Don't need to update in this PR. We need to revisit ```handlers.js``` and fix syntax issues. 
- A mix of ```"``` and ```'```.
- Remove unused variables e.g., ```res```
- ```const```/```let``` over ```var```